Documentation/Blazorise.Docs/Components/TopMenuEnd.razor

```diff
@@ -8,8 +8,12 @@
     <BarLink To="account">Account</BarLink>
 </BarItem>
 <BarItem>
-    <BarLink To="https://github.com/Megabit/Blazorise" Target="Target.Blank">
-        <Icon Name="IconName.Star" IconStyle="IconStyle.Regular" IconSize="IconSize.Large" TextColor="TextColor.Warning" />
-        Star us on GitHub
+    <BarLink Class=""  To="https://github.com/Megabit/Blazorise" Target="Target.Blank">
+        <div class="d-flex d-md-block d-xl-flex align-items-center">
```

Comment:
I did that, but I had to use also the `Class="d-xl-flex"` as I am not sure how to "force" it in Flex utility as it only provides row, column, reverseRow/Column options...

Reply:
Which branch are you targeting? Based on your branch name, I would say 1.6. but it is currently on `master`.

Anyway, it is best to make it on master because I have made some changes to the Flex utility to allow the definition of default breakpoints.

This is how it should look like:

`<Div Display="Display.Flex.Row.Block.OnDesktop" Flex="Flex.Default.OnWidescreen.Flex.AlignItems.Center">`

Reply:
yeah, it's 1.6 to master. I will do it master to master.